presto-main/src/test/java/com/facebook/presto/operator/scalar/TestStringFunctions.java

```diff
@@ -523,6 +696,16 @@ public void testUpper()
     }
 
     @Test
+    public void testCharUpper()
+    {
+        assertFunction("UPPER(CAST('' AS CHAR(10)))", createCharType(10), "");
+        assertFunction("UPPER(CAST('Hello World' AS CHAR(11)))", createCharType(11), "HELLO WORLD");
+        assertFunction("UPPER(CAST('what!!' AS CHAR(6)))", createCharType(6), "WHAT!!");
+        assertFunction("UPPER(CAST('\u00D6sterreich' AS CHAR(10)))", createCharType(10), upperByCodePoint("\u00D6STERREICH"));
```

Comment:
Use lower-case o-umlaut as test input for UPPER function in `testUpper` and `testCharUpper`